src/main/scala/org/graphframes/lib/ConnectedComponents.scala

```diff
@@ -308,8 +308,6 @@ object ConnectedComponents extends Logging {
     val g = prepare(graph)
     val vv = g.vertices
     var ee = g.edges.persist(intermediateStorageLevel) // src < dst
-    val numEdges = ee.count()
```

Comment:
This count is needed to materialize previously persisted edges DataFrame. Why is it considered as unnecessary? cc: @SauronShepherd